analyzers/src/SonarAnalyzer.VisualBasic/Rules/UseReturnStatement.cs

```diff
@@ -0,0 +1,64 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2020 SonarSource SA
```

Comment:
We've updated the license headers meanwhile
```suggestion
 * Copyright (C) 2015-2021 SonarSource SA
```